app/models/location.py

```diff
@@ -123,8 +123,79 @@ def generate_fake(count=10):
                 street_address=fake.street_address(),
                 city=fake.city(),
                 state=fake.state(),
-                zip_code=choice(zip_codes)
             )
+            a.zip_code = choice(zip_codes)
+            db.session.add(a)
+            db.session.commit()
+
+    @staticmethod
+    def generate_real_fake():
```

Comment:
yeah, sorry, missed that. Let's just get rid of that one and change the name of this one.